### DISCUSSION TITLE
Excluding gzipped files

### DISCUSSION THREAD

**Discussion by davidsalazar** [opcodesio/log-viewer#360] [Q&A] [answered]
Looks like this doesn't read gzipped files that are rotated and gzipped. What's the best way to exclude those files? I tried adding .gz to the exclude array but I'm assuming that glob match doesn't quite work? 

I did search discussions and found this, but no solution. https://github.com/opcodesio/log-viewer/discussions/138

**Comment by arukompas** [comment, marked as answer]
How did you define it?

```
'exclude' => [
  '*.gz',
]
```

Above should work for the laravel's storage/logs folder. 

Keep in mind that the include/exclude definitions are scoped to the storage/logs folder unless you use absolute paths (start with a forward slash)

**Reply by davidsalazar** [reply]
thanks. I actually just changed includes from /var/log/nginx/* to /var/log/nginx/*.log

Looked into the codebase to see how this was working. Thanks.
